dot-studio/golang

```diff
@@ -29,44 +29,40 @@ DOC
 function setup_go_workspace() {
   # Verify that we are using the Go Scaffolding
   if [[ ! "$pkg_scaffolding" =~ "scaffolding-go" ]]; then
-    echo "ERROR: Unable to setup Go Workspace."
-    echo "${FUNCNAME[0]}: requires the 'pkg_scaffolding' variable to exist in your plan.sh"
+    error "Unable to setup Go Workspace."
+    error "${FUNCNAME[0]}: requires the '$(yellow pkg_scaffolding)' variable to exist in your plan.sh"
     return 2
-  fi
+  fi;
 
   # Verify if the Go Workspace has been already configured
-  if [[ $scaffolding_go_workspace_src  ]]; then
+  if [[ -n $scaffolding_go_workspace_src ]]; then
     return 0
   fi
 
   # Currently the scaffolding-go has all the logic to setup the
   # Go Workspace, lets leverage all its functionality
-  echo "=> Configuring Go Workspace. [please wait]"
-  export scaffolding_go_gopath="${scaffolding_go_gopath:-/hab/cache/src/go}"
+  log_line "${YELLOW}Configuring Go Workspace${NC}"
+
+  export scaffolding_go_gopath="${scaffolding_go_gopath:-/hab/cache/src/scaffolding-go-gopath}"
```

Comment:
Is that a problem changing the default gopath?

Reply:
Nope, I am just updating it from what the core scaffolding is doing right now: https://github.com/habitat-sh/core-plans/blob/master/scaffolding-go/lib/scaffolding.sh#L32

The only reason I had to duplicate this line is because in the context of the studio we don't have that `HAB_CACHE_SRC_PATH` variable, but now that you mentioned, I think we can just export it so we don't have to do this duplication! Lets talk about it on Monday!!!